### PR TITLE
pi-manage can cleaup old entries in the authcache

### LINKED ISSUE #1471: Script for deleting expired entries from the auth cache
Similarly to the user cache, we need a script that deletes outdated entries from the auth cache. See #1466

### PR DESCRIPTION
**Pull request by cornelinux**
Closes #1471

### CODE REVIEW

File: pi-manage [hunk 1]
```diff
@@ -559,6 +562,16 @@ except TypeError:
     profile = manager.command(profile)
 
 
+@authcache_manager.command
+def cleanup(minutes=480):
+    """
+    Remove entries from the authcache which last_auth entry is older than
```
Comment (plettich) [inline pi-manage]: which -> where

Comment (cornelinux) [reply pi-manage]: You see, I think I wanted to say "for which..."

Comment (cornelinux) [reply pi-manage]: done in 4506a7bb

**Comment by cornelinux** [comment]
Hi @plettich 
we need this pull request in branch 2.23 for 2.23.5.
So we can not wait for #1472.

But: We can merge this (1480) into master *after* the PR #1472 was merged.
But we should merge this one into branch-2.23 like now.

Thanks a lot.

**Comment by plettich** [comment]
I should look at the branches to merge and not assume it is always master...

**Pull request merged**